site/en/docs/android/custom-tabs/howto-custom-tab-native-apps/index.md

```diff
@@ -37,9 +37,18 @@ static boolean launchNativeApi30(Context context, Uri uri) {
 ```
 
 ## Before Android 11
+
 Even though the application may target Android 11, or API level 30, previous Android versions will
-not understand the `FLAG_ACTIVITY_REQUIRE_NON_BROWSER` flag, so we need to resort to querying the
-Package Manager in those cases:
+not understand the `FLAG_ACTIVITY_REQUIRE_NON_BROWSER` flag. To support those devices, we can query the Package Manager. Our approach is to  query the Package Manager for applications that support a generic "http" intent. Those applications are likely browsers.
```

Comment:
since it didn't break any linting, i'm assuming the long line is not a problem? wasn't sure if we were expected to match a max line length

Reply:
afaik not for markdown

Reply:
yeah, def not markdown. just meant for the project. style guide or whatnot.